src/card.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC, PropsWithChildren } from 'react';
+
+import clsx from 'clsx';
+
+/**
+ * Renders a padded card with white background and drop shadow. "isFlat" uses
+ * a yellow background and no shadow instead. Children are placed in a
+ * 1-column grid.
+ */
+export const Card: FC<PropsWithChildren<{ id?: string; isFlat?: boolean }>> = ({
+  id,
+  isFlat,
+  children,
+}) => (
+  <div
+    id={id}
+    className={clsx(
+      'rounded-xl',
+      'overflow-clip',
+      'min-w-52',
```

Comment:
I think currently this is fine as-is, but in other React things I've done recently I've made a move to keep sizing out of the default styling or at least to allow overriding it from the parent, to avoid over-fitting to the initial use-case for the component. Just a note not a blocker for this PR.